exporting/README.md

```diff
@@ -252,7 +252,8 @@ Configure individual connectors and override any global settings with the follow
     within each pattern). The patterns are checked against both chart id and chart name. A pattern starting with `!`
     gives a negative match. So to match all charts named `apps.*` except charts ending in `*reads`, use `!*reads
     apps.*` (so, the order is important: the first pattern matching the chart id or the chart name will be used -
-    positive or negative).
+    positive or negative). There is also a URL parameter `filter` that can be used while querying `allmetrics`. The URL
+    parameter has a higher priority than the configuration option.
```

Comment:
This feature is too good to hide it here. I suggest:

- reformat the "Features" section: use `###` (level 3 heading) instead of ordered list.
- add this info to "Netdata can filter metrics" (current 2.)